src/meta/processors/zone/AddHostsIntoZoneProcessor.cpp

```diff
@@ -98,7 +99,10 @@ void AddHostsIntoZoneProcessor::process(const cpp2::AddHostsIntoZoneReq& req) {
   zoneHosts.insert(zoneHosts.end(), hosts.begin(), hosts.end());
   data.emplace_back(std::move(zoneKey), MetaKeyUtils::zoneVal(std::move(zoneHosts)));
 
-  HostInfo info(0, cpp2::HostRole::STORAGE, "");
+  auto now = time::WallClock::fastNowInMilliSec();
```

Comment:
Do we really need to write a `hostKey` here? If the host is alive, it will report to meta by heartbeat later, if not alive, there won't be any heartbeat.

Previously we only write this key in `ActiveHostsMan`.